### PR TITLE
Better management of cached script in waitForScript.

### PR DESCRIPTION
**Pull request by stevematney**

### CODE REVIEW

**Review by JosephDSchwartz — APPROVED** [review]

File: src/scriptCache.ts [hunk 1]
```diff
@@ -94,3 +94,59 @@ export function removeScriptUpdater(
     (currentUpdater) => currentUpdater !== updater
   );
 }
+
```
Comment (borland-david) [inline src/scriptCache.ts:97]: Not your changes, but do you want to fix the spelling above `getCahcedScriptUpdaters`?

**Review by borland-david — APPROVED** [review]

**Comment by stevematney** [comment]
This PR is part of this prerelease version for testing: 1.4.2-promise-consolidation-94cbae-5.1

You can test it by using:
```bash
npm install scriptloader-component@1.4.2-promise-consolidation-94cbae-5.1
```

**Comment by stevematney** [comment]
I've confirmed locally that this is working. Merging now.

**Pull request merged**